src/main/java/dukes/util/UI.java

```diff
@@ -0,0 +1,235 @@
+package dukes.util;
+
+import dukes.task.Task;
+
+import java.util.Scanner;
+
+/**
+ * The util class for reading inputs and providing feedbacks to user.
+ */
+public class UI {
+    // mainly use static methods
+    // need to handle the exceptions raised by Parser
+    public static final String WORD_DIVISHION_LINE = "________________________________________";
```

Comment:
I think this might be a typo, might want to spell check division for clarity. But otherwise, I like the use of the constants. Very clean and neat looking.